### DISCUSSION TITLE
Can you tell?

### DISCUSSION THREAD

**Discussion by Gouravyadavyt** [NXTS-Developers/MOFHY-Lite#78] [General]
Hello friends I recently created a mofh template for my site can give some idea of makes it better

url - https://easyteachhost.tk

**Comment by greenreader9** [comment]
You need an SSL certificate.

>Attackers might be trying to steal your information from easyteachhost.tk (for example, passwords, messages, or credit cards). [Learn more](chrome-error://chromewebdata/#)

**Comment by mahtab2003** [comment]
Page layout is same in all devices. You should use different layouts for different devices

**Reply by Gouravyadavyt** [reply]
SSL is enabled
I make sure that it's is also mobile friendly.

**Comment by mahtab2003** [comment]
I'm my phone the layout is messed up

**Reply by Gouravyadavyt** [reply]
I don't want to share my personal things

**Reply by Gouravyadavyt** [reply]
Can I register my company right now

**Reply by mahtab2003** [reply]
Company?

**Reply by Gouravyadavyt** [reply]
Mahtab sir I change my site layout check please

**Reply by mahtab2003** [reply]
Nice but pop up ads?

**Reply by Gouravyadavyt** [reply]
Adsense are not give me approval 
Just Because I used propeller ads

**Reply by mahtab2003** [reply]
Oh
